besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -696,6 +697,11 @@ static class PrivacyOptionGroup {
       description = "Specifies the number of last blocks to cache  (default: ${DEFAULT-VALUE})")
   private final Integer numberOfblocksToCache = 0;
 
+  @CommandLine.Option(
+      names = {"--cache-precompiles"},
+      description = "Specifies whether to cache precompile results (default: ${DEFAULT-VALUE})")
+  private final Boolean enablePrecompileCaching = true;
```

Comment:
Can we disabled on default and enabled on demand, for a performance profile on block processing ?

Reply:
I enabled it by default, with or without parallel processing.  Do you want to make it disabled by default?

Reply:
Yes in the first place, we can make it default after some time of running it.

Reply:
I understand that the feature will be enabled either with the parallelization flag, or when `--cache-precompiles` is set to true, correct ? 
So basically, even if `--cache-precompiles=false`, we can still have precompile caching enabled. I think it can be confusing for the users. Let's make it simple. 
I think `--cache-precompiles` should be the only flag that can define if this feature will enable it or not.